### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by xepozz**
image

### CODE REVIEW

**Comment by mamadza** [comment]
Hi, I have question, what's the reason to use angular? 
If angular version goes higher what's happen to incompatible

**Comment by xepozz** [comment]
> Hi, I have question, what's the reason to use angular?

After some private discussions I've realized that Angular will be more suitable for our frontend apps.

> If angular version goes higher what's happen to incompatible

Incompatible for what?

**Comment by samdark** [comment]
> Incompatible for what?

Incompatible with the application using another version of Angular. Because debug frontend consists of two parts:

1. Viewer that is separate app. No problem here.
2. Toolbar that is displayed at the bottom of application pages. This may be a problem.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~10.0.3",
+    "@angular/common": "~10.0.3",
+    "@angular/compiler": "~10.0.3",
+    "@angular/core": "~10.0.3",
+    "@angular/forms": "~10.0.3",
+    "@angular/http": "~7.2.16",
```
Comment (rustamwin) [inline package.json:19]: https://angular.io/guide/deprecations#angularhttp

Comment (xepozz) [reply package.json:19]: Anchor isn't found

**Comment by xepozz** [comment]
> Incompatible with the application using another version of Angular.

I get it. 
Toolbar may be implemented as a extension to another JS frameworks (not only for Angular). 
The viewer will separated standalone application, it won't used as a framework extension.

**Comment by samdark** [comment]
OK. Let's start with this one.

**Pull request merged**